mmv1/products/gkeonprem/VmwareNodePool.yaml

```diff
@@ -120,12 +120,14 @@ properties:
         description: |
           The OS image to be used for each node in a node pool.
           Currently `cos`, `ubuntu`, `ubuntu_containerd` and `windows` are supported.
+        default_from_api: true
```

Comment:
`default_from_api` is not meaningful if the field is marked `required` - you'd need to mark it as not required.